README.md

```diff
@@ -68,3 +68,61 @@ function SaveButton() {
     return <Button>Hello World</Button>;
 }
 ```
+
+<br />
+
+## Linking and Unlinking for Local Packages
+
+At times it may be required to link an external pacakge to a local version of the admin ui for testing. To do this have 2 terminals open. One terminal should be in the root of your local folder of the admin ui package and the other should be in the root to the project you would like to use the local package in. VS code should also be open.
```

Comment:
@NoahThomlison typo on pacakge. Also should this read more like `At times in may be required to link a project to admin-ui locally for development testing purposes...` not sure if "an external package" makes a ton of sense